resume.md

```diff
@@ -1,49 +1,46 @@
-* ...
-
-## AWARDS/ACKNOWLEDGEMENTS
-* **COLLEGE Alumni Scholarship Recipient** - Alumni Scholarship Committee... *Month 20XX*
-* **1st Place Best Hack** - Hackathon Event... *Month 20XX*
-
-## Events/Conferences
-* **Cal Hacks 7.0 (remote)** - Berkeley, CA ... *Month 20XX*
-* **Cal Hacks 6.0** - Berkeley, CA ... *Month 20XX*
-* **WWDC 2019** - San Jose, CA ... *Month 20XX*
+Objective
+--------------------------------------------------------
+To obtain an entry-level position in software engineering in order to obtain industry experience, expand my knowledge, and maximize my skillset.
+
+Education
+--------------------------------------------------------
+University of Southern California, Los Angeles CA	Aug 2017 – June 2021 (Expected)
+Bachelor of Science, Mechanical Engineering
+Minor in Computer Programming
+GPA: 3.47, Dean’s List: 2 semesters	
+Computer Science Coursework: Programming Systems Design  •  Analysis of Algorithms
+
+Programming Skills
+--------------------------------------------------------
+Languages: C++  •  Java •  HTML  •  CSS  •  PHP  •  JavaScript  •  SQL  •  MATLAB
+Technologies: React •  Bootstrap  •  MongoDB  •  AWS  •  Git  •  Siemens NX  
+
+Experience
+--------------------------------------------------------
+Teaching Assistant	Aug 2019 – present
+USC Viterbi School of Engineering	Los Angeles, CA
+•	Assisted two professors for Object Oriented Programming and Professional C++
+•	Administered weekly office hours which improved debugging and communication skills
+
+Enterprise Applications Intern	June 2019 – Oct 2019
+Los Angeles Unified School District	Los Angeles, CA
+•	Consulted the Information Technology Desk Department and managed their pages using HTML, CSS, and JS to ensure responsive design and ADA compliancy
+•	Established the LAUSD hall of fame of 2,000+ alumni by implementing a web form using HTML, CSS, and PHP to input data into a SQL Server database
+
+Academic Projects	
+--------------------------------------------------------
+Sorting Algorithm Visualizer/Timer	July 2020 – July 2020
+•	Created an app to visualize/time merge, heap, quick, bubble, selection, and insertion sort
+•	Developed a data table to compare the run times of algorithms depending on sample size
+•	Technologies: React, JavaScript
+
+NASA Images	Jun 2020 – July 2020
+•	Created a MERN stack application to cleanly display image archives from NASA’s Astronomy Photo of the Day and Mars Rover API
+•	Implemented a membership feature using MongoDB to allow users to save images to a list
+•	Technologies: React, MongoDB, Express, NodeJS, AWS
+
+Smart Aquarium	Aug 2019 – Dec 2019
+•	Sponsored by Nuhzdin Lab at USC to build a system of tanks to simulate ocean conditions
+•	Collaborated with three teammates to develop an Arduino monitoring system to ensure that tank conditions will prolong the lifespan of kelp samples from 3 hours to 3 days
```

Comment:
Can you be specific on the tools you used to make this possible?  use any interesting sensors?

How did you use these technologies to help your accomplish your project?